src/hotspot/share/gc/shared/generation.hpp

```diff
@@ -137,6 +137,7 @@ class Generation: public CHeapObj<mtGC> {
                                         // generation can currently hold.
   virtual size_t used() const = 0;      // The number of used bytes in the gen.
   virtual size_t free() const = 0;      // The number of free bytes in the gen.
+  virtual size_t live() const = 0;
```

Comment:
Needs a comment to match the lines above? Say, `// The estimate of live bytes in the gen.`

Reply:
👍